src/helpers/checkMaintenance.js

```diff
@@ -1,11 +1,12 @@
 // @flow
-import moment from "moment";
+import { formatDistance } from "date-fns";
+import { getCurrentUnixTime } from "./DateUtil";
 import { getMaintenanceCaption } from "../Domain/Maintenance";
 
 export default function checkMaintenance(maintenance: ?number): string {
     if (!maintenance) {
         return getMaintenanceCaption("off");
     }
-    const delta = maintenance - moment.utc().unix();
-    return delta <= 0 ? getMaintenanceCaption("off") : moment.duration(delta * 1000).humanize();
+    const delta = maintenance - getCurrentUnixTime();
+    return delta <= 0 ? getMaintenanceCaption("off") : formatDistance(0, delta * 1000);
```

Comment:
But in cases like this I'll prefer on the contrary wrap code to functions

Reply:
Because it's difficult to understand, what means `* 1000`

Reply:
Let's write some function like `parseUnixTimestampToJS`

Reply:
sure i'll do that : )